packages/story-editor/src/components/library/panes/media/local/hotlink/hotlinkModal.js

```diff
@@ -106,6 +106,7 @@ function HotlinkModal({ isOpen, onClose }) {
       onPrimary={() => onInsert()}
       primaryText={__('Insert', 'web-stories')}
       secondaryText={__('Cancel', 'web-stories')}
+      primaryRest={{ disabled: errorMsg || !link }}
```

Comment:
That's a terrible prop name! Is it used in many places? It really ought to be called `primaryProps` or even better `primaryButtonProps`.

Reply:
It's used in quite a few places, yes. There's also `secondaryRest` for the secondary button.

Reply:
I've filed #8980.